### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by gentilsol** [balzack/databag#41] [Ideas]
can you create a radmap where you can see how the project is going on

**Comment by balzack** [comment]
Lately I just maintain a large todo list. Is this what you are referring to, or something more formal in github?

**Reply by CoffeBlock** [reply]
I think he meant a roadmap in GitHub 
- https://github.com/orgs/github/projects/4247/views/1

It will require you to create a project in GitHub and use the Roadmap template. If you have a private project, maybe you should make it public. 

(Note: If you have a team of devs, you might want to create an org and a project inside it.)

**Reply by balzack** [reply]
Thanks for the link. That view makes a lot more sense to me. I started the effort but am new to github projects, and the default view didn't seem to represent a roadmap. For now I just maintain the road map as a list with time estimates, and was hoping for something similar.

**Reply by CoffeBlock** [reply]
I can help you set up the roadmap, though if there are other more important things to do in this project, I am happy to help!

**Comment by balzack** [comment]
I'll take another pass at the roadmap since it's currently a high-level todo list. I probably will be asking for help if you've done this before.

Currently my main focus for the last couple months and the next couple months is creating an SDK and refactoring the current messenger to use that SDK. The core logic (SDK) is a decentralized data sharing network, that I hope to apply to apps beyond a messenger. 

During that time I can push out small feature updates and bug fixes, but larger feature I plan to wait until the SDK refactor is done.

**Reply by CoffeBlock** [reply]
Alright, I am more than happy to help if I can!
